.github/workflows/workflow.yml

```
@@ -0,0 +1,25 @@
+name: Python CI
+
+on:
+  push:
```

Wollen wir hier wirklich on push lassen oder lieber?
```suggestion
on:
  pull_request:
    branches:
    - main # Runs on PRs targeting the main branch

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
```

---

Ne, kann ruhig immer during dev getestet werden.